src/tree.rs

```diff
@@ -485,7 +485,7 @@ pub mod testing {
             0x9c, 0x52, 0x7f, 0x0e,
         ];
 
-        let mut frontier = BridgeFrontier::<MerkleCrhOrchardOutput, 32>::new();
+        let mut frontier = BridgeFrontier::<MerkleCrhOrchardOutput, 32>::empty();
```

Comment:
This doesn't seem to belong to this PR.

Reply:
This is related to a change that has been merged in the incrementalmerkletree crate. Implementing `new` requires implementing `Default` and I think `Default` is a bad idea in all cases.